src/components/pages/TechnicalUserManagement/TechnicalUserTable.tsx

```diff
@@ -89,6 +98,20 @@ export const TechnicalUserTable = () => {
     },
   ]
 
+  const renderStatusColor = (status: string) => {
+    if (status.toLowerCase() === ServiceAccountStatus.ACTIVE.toLowerCase())
+      return 'confirmed'
+    else if (
+      status.toLowerCase() === ServiceAccountStatus.INACTIVE.toLowerCase()
+    )
+      return 'declined'
+    else if (
+      status.toLowerCase() === ServiceAccountStatus.DELETED.toLowerCase()
+    )
+      return 'deleted'
+    else return 'pending'
+  }
+
```

Comment:
the type ServiceAccountStatus has a number of values - so best is to create a map

```
const statusColorMap: Record<ServiceAccountStatus, string> = {
  ServiceAccountStatus.ACTIVE: 'confirmed',
  ServiceAccountStatus.INACTIVE: 'declined',
  ServiceAccountStatus.DELETED: 'deleted',
  ServiceAccountStatus.PENDING: 'pending',
  ServiceAccountStatus.PENDING_DELETION: 'pending',
}
```